### ISSUE TITLE
Incompatible with `Next.js`.

### ISSUE THREAD

**Issue by rommelmamedov** [Upmostly/modali#49]
**Error**: "Global CSS cannot be imported from within node_modules." error in Next.js

It's because of CSS import in main `index.js` file.

https://github.com/upmostly/modali/blob/1a1f58a70167dda11f4a2b8492bd4a0bcdb4fc92/src/index.js#L6

**Comment by darwinshameran** [comment]
Using [`@zeit/next-css`](https://github.com/vercel/next-plugins/tree/master/packages/next-css) yields error:

    .modali-overlay {
    ^


    SyntaxError: Unexpected token '.'
        at wrapSafe (node:internal/modules/cjs/loader:1018:16)
        at Module._compile (node:internal/modules/cjs/loader:1066:27)
        at Object.Module._extensions..js (node:internal/modules/cjs/loader:1131:10)
        at Module.load (node:internal/modules/cjs/loader:967:32)
        at Function.Module._load (node:internal/modules/cjs/loader:807:14)
        at Module.require (node:internal/modules/cjs/loader:991:19)
        at require (node:internal/modules/cjs/helpers:92:18)

**Comment by atefBB** [comment]
@rommelmamedov Did you solve this issue? Any help can be appreciated!

**Comment by ramilmamedoff** [comment]
> @rommelmamedov Did you solve this issue? Any help can be appreciated!

No, I just used [React-Modal](https://www.npmjs.com/package/react-modal) :)
